templates/emails/thankyou.fr.hbs

```diff
@@ -1,7 +1,7 @@
 {{#if interval }}
-Subject: Merci pour votre donation de {{currency order.totalAmount currency=order.currency}}/{{interval}} à {{collective.name}}
+Subject: Merci pour votre contribution de {{currency order.totalAmount currency=order.currency}}/{{interval}} à {{collective.name}}
```

Comment:
Changing the wording there, `donation` was not appropriate. `contribution` matches our gloassary.